docker/Dockerfile-workers

```diff
@@ -16,21 +16,20 @@ RUN git clone https://github.com/operator-framework/operator-registry $OPM_SRC &
     (cd $OPM_SRC && git checkout $OPM_TAG && make build) && \
     cp $OPM_SRC/bin/opm /usr/bin/opm
 
-FROM centos:8
```

Comment:
Don't we want to develop against CentOS 8 since that is the OS we deploy IIB on? Otherwise, we could end up using newer skopeo/podman features without realizing that they don't work in the RPMs present in CentOS 8.

Reply:
Yeah... I couldn't make it work with centos at all :disappointed:

Reply:
Finally got this to work with `centos:8`. It turns out that the changes I had to do in the Dockerfile also enabled this.